sample-app/app/controllers/application_controller.rb

```diff
@@ -1,8 +1,9 @@
 class ApplicationController < ActionController::Base
+  include SessionsHelper
   protect_from_forgery with: :exception
   before_action :set_locale
   
-  private
+  private 
```

Comment:
```suggestion
  private
```
Xóa dấu cách sau private em ơi